### PR TITLE
Add Myself and A-Chancey to comment-commands.yml

### PR DESCRIPTION
**Pull request by ANickelN**
#### Summary
None

#### Purpose of change

Add myself and another contributor to the list of those able to /confirm issues.

#### Describe the solution

Add our names to the list

#### Describe alternatives you've considered

Coming up with an alternative confirmation system, but why bother when this is easier?

#### Testing

No testing

### CODE REVIEW

**Comment by Maleclypse** [comment]
@a-chancey Confirming here that you want to be added to the list and that your id is spelled correctly in the change?

**Comment by Cimanyd0** [comment]
I would also like to be added, if that's ok with @Maleclypse.

**Comment by sonphantrung** [comment]
Just for some curiosities: What are the requirements to get myself added into that? Or can I just randomly add me into?

**Comment by a-chancey** [comment]
Looks correct to me

**Comment by Night-Pryanik** [comment]
> What are the requirements to get myself added into that?

No strict requirements. Be active in discussions, test issues that require confirmations, and someday you might get this ability.

File: .github/comment-commands.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ users:
     - Inglonias
     - GuardianDll
     - hexagonrecursion
+    - ANickelN
+    - a-chancey
```
Comment (Maleclypse) [inline .github/comment-commands.yml:27]:
```suggestion
    - a-chancey
    - Cimanyd0
```

**Comment by Maleclypse** [comment]
> I would also like to be added, if that's ok with @Maleclypse.

Not a problem :)

**Pull request merged**